crates/router/src/core/configs.rs

```diff
@@ -0,0 +1,18 @@
+use crate::{
+    core::errors::{self, utils::StorageErrorExt, RouterResponse},
+    db::StorageInterface,
+    services::ApplicationResponse,
+    types::storage,
+};
+
+pub async fn read_config(
+    store: &dyn StorageInterface,
+    key: &str,
+) -> RouterResponse<storage::Config> {
```

Comment:
Please ensure that the type returned by the API is an API model and not a storage model.